edbdeploy/data/ansible/kubernetes_roles/setup_cloudnativepostgres_sandbox/README.md

```diff
@@ -0,0 +1,91 @@
+# setup_cloudnativepostgres_sandbox
```

Comment:
Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/setup_cloudnativepostgres_sandbox